homeassistant/components/sonos/__init__.py

```diff
@@ -12,7 +12,7 @@
 from urllib.parse import urlparse
 
 from aiohttp import ClientError
-from requests.exceptions import Timeout
+from requests.exceptions import RequestException, Timeout
```

Comment:
```suggestion
from requests.exceptions import Timeout
```
Importing and catching `RequestException` is unnecessary as we're already catching `OSError`. Here you can see that `RequestException` is a subclass of `IOError`, which has been merged into `OSError`:
https://github.com/psf/requests/blob/7f694b79e114c06fac5ec06019cada5a61e5570f/requests/exceptions.py#L12

See version change notes for `OSError` here: https://docs.python.org/3/library/exceptions.html#OSError

Reply:
Excellent point, thank you.  On unrelated note, I found another issue in this routine when I did extensive system testing - plugging and unplugging speakers - simple fix though it required more unit tests - will have that change and this tomorrow.

Reply:
Fixed